packages/conf-bmake/conf-bmake.1.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "michipili@gmail.com"
+homepage: "http://www.crufty.net/help/sjg/bmake.htm"
+authors: "sjg@crufty.net"
+bug-reports: "sjg@crufty.net"
+license: "BSD"
+config-state-vars: [
+  path "The detected path a BSD Make compatible make program."
+]
+build: [
+  ["sh" "%{build}%/files/detect_program.sh" "%{prefix}%/lib/conf-bmake" "bmake"] {!(os = "freebsd" | os = "openbsd" | os = "netbsd")}
+  ["sh" "%{build}%/files/detect_program.sh" "%{prefix}%/lib/conf-bmake" "make"] {(os = "freebsd" | os = "openbsd" | os = "netbsd")}
```

Comment:
Pins are intended to be used on source repositories not metadata repositories. Your `pin` is mixing the two types of repository. I recommend you either make a source repository to hold the shell script and treat this like a normal (but very minimal) opam package or forgo the use of pins and simply run `opam upd` when you would like to test the package.